api-docs/cloud-serverless/ref.yml

```diff
@@ -14002,26 +14011,31 @@ paths:
           - [Best practices for optimizing writes](/influxdb/cloud-serverless/write-data/best-practices/optimize-writes/)
         required: true
       responses:
-        '204':
+        "204":
           description: |
-            Success. Data is written and queryable.
-        '400':
+            Success. All points in the batch are written and queryable.
+        "400":
           content:
             application/json:
               examples:
-                measurementSchemaFieldTypeConflict:
-                  summary: field type conflict thrown by an explicit bucket schema
+                partialWriteErrorWithRejectedPoints:
```

Comment:
Perfect. No change for `partialWriteErrorWithRejectedPoints`. (Because is NOT configurable for serverless.)

Reply:
Seperately, can we have 2 separate examples so the user can delineate:
* per-line rejection (with something writtten). 
* not per-rejection. Nothing written. Error message is NOT `partialWriteErrorWithRejectedPoints`. e.g. namespace errors

Reply:
Created separate issue.